src/puppetlabs/puppetdb/export.clj

```diff
@@ -27,22 +27,25 @@
 ;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
 ;;; Utility Functions
 
+(def file-suffix ".json")
+
 (defn export-filename
   "For anything we store historically and need unique names"
   [{:keys [certname producer_timestamp]}]
   (let [formatted-start-time (->> producer_timestamp
                                   time-coerce/to-date-time
                                   (time-fmt/unparse (time-fmt/formatter "yyyy-MM-dd'T'HH:mm:ss.SSS'Z'")))]
```

Comment:
We don't care about the formatting, but it needs to be normalized so that it is always the same regardless of input format. Calling generic-identity-hash will perform the same conversion as above though, so I can switch it and avoid having that date formatting in two places.